src/libstd/primitive_docs.rs

```diff
@@ -165,6 +165,27 @@ mod prim_char { }
 ///
 mod prim_unit { }
 
+#[doc(primitive = "ref")]
+//
+/// Borrowed references, `&T`, and `&mut T`.
+///
+/// References fundamental to Rust's system of ownership and borrowing. They
+/// are represented as pointers, but the compiler can statically ensure that no
+/// unsafety results from passing and dereferencing them.
+///
+/// Shared references (`&T`) allow read-only access to the pointee, while
```

Comment:
This is somewhat nuanced where types like `&Cell<T>` are actually read/write, not just read only. Perhaps this could just indicate "allows aliasing"?